src/Agent/NewRelic/Profiler/MethodRewriter/Instrumentors.h

```diff
@@ -29,6 +29,7 @@ namespace NewRelic { namespace Profiler { namespace MethodRewriter
     {
         bool Instrument(IFunctionPtr function, InstrumentationSettingsPtr instrumentationSettings) override
         {
+            instrumentationSettings->GetInstrumentationConfiguration()->CheckForEnvironmentInstrumentationPoint();
```

Comment:
It would be nice if we did not need to make this call every time, but that problem can be solved later.

Reply:
Agreed. I briefly tried to find which internal method was setting it, so we could key off that, but no luck. As we do more testing I'm hoping we can find a cleaner place for it.